spec/requests/user/user_show_spec.rb

```diff
@@ -62,6 +59,16 @@
       )
     end
     # rubocop:enable RSpec/ExampleLength
+
+    it "does not render a key if no value is given" do
```

Comment:
Does anyone have an opinion on adding/not adding `:aggregate_failures` here?

Reply:
We're definitely not consistent in using `:aggregate_failures`. I', good either way :D